### PR TITLE
Install basic package.xml for rosdep resolve

### PR DESCRIPTION
**Pull request by paulbovbel**

### CODE REVIEW

**Review by AadityaRavindran — APPROVED** [review]

File: package.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+  <license>MIT</license>
+  <buildtool_depend>cmake</buildtool_depend>
+
+  <!-- <depend>build-essentials</depend> -->
+
+  <!-- The export tag contains other, unspecified, tags -->
+  <export>
+    <build_type>cmake</build_type>
+    <pip_requirements>requirements.txt</pip_requirements>
```
Comment (AadityaRavindran) [inline package.xml]: Why pip?

File: package.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>cpp-statsd-client</name>
+  <version>1.0.0</version>
```
Comment (AadityaRavindran) [inline package.xml]: The latest released version upstream is 2.0.1

File: package.xml (hunk 1 above)
Comment (garyservin) [inline package.xml]: I don't think we need this

**Review by garyservin — APPROVED** [review]

File: package.xml (hunk 1 above)
Comment (paulbovbel) [reply package.xml]: copy-paste, thanks!

File: package.xml (hunk 2 above)
Comment (garyservin) [reply package.xml]: yea we should match this to the upstream version, otherwise I'll have to do it once we go to release a new version :laughing:

**Pull request merged**